docs/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,212 @@
+the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu.
+Follow our [guide to using Connectors](/components/connectors/use-connectors/index.md) to learn more.
+
+## Connection
+
+Enter your SOAP service URL in the field **Service URL**, for example `https://myservice.com/service/MyService.wso`.
+
+## Authentication
+
+Select the authentication type from the **Authentication** dropdown.
+
+### None
+
+Use **None** if the SOAP service does not require authentication.
+
+### WSS username token
+
+Use **WSS username token** in the **Authentication** dropdown when the requested SOAP endpoint requires
+[username token extension](https://docs.oasis-open.org/wss/v1.1/wss-v1.1-spec-pr-UsernameTokenProfile-01.htm#_Toc104276211).
+
+Enter **Username**, **Password**, and indicate if the password is encoded.
+
+:::note
+The **SOAP Connector** currently supports only `SHA-1` password encoding.
+:::
+
+### WSS signature
+
+Use the **WSS signature** in the **Authentication** dropdown when the requested SOAP endpoint requires a message to be
+cryptographically signed with a [signature](http://docs.oasis-open.org/wss-m/wss/v1.1.1/cs01/wss-SOAPMessageSecurity-v1.1.1-cs01.html#_Toc307407954).
+
+Enter all necessary fields according to your service specification.
+
+## SOAP message
+
+### SOAP version
+
+Select the latest supported version of the SOAP service.
+
+### SOAPAction HTTP header
+
+Enter the SOAPAction HTTP header that will be used in the request. Leave this value blank if the SOAPAction HTTP header
+won't be used in your request.
+
+### SOAP header
+
+From the dropdown, select whether the **SOAP header** is required, and if so, in which format you wish to provide it.
+
+### SOAP body
+
+From the **SOAP body** dropdown, select whether you will provide the SOAP request body in a form of **Template**, or
+**XML compatible JSON**.
+
+#### Template
+
+When **Template** is chosen, enter the **XML template** value, for example `<camunda:Param><camunda:ParamType>{{paramValue}}</camunda:ParamType></camunda:Param>`.
+
+Enter the **XML template context** value, for example `={paramValue: 1234567890}`, and enter the **Namespaces** value, for example `={"camunda":"http://my.service.com/webservicesserver/"}`.
+
+#### XML compatible JSON
+
+When **XML compatible JSON** is chosen, enter the **JSON definition**, for example
+
+```xml
+<camunda:Object01>
+  <camunda:Object02>{{myObjectValue}}</camunda:Object02>
+</camunda:Object01>
+```
+
+Enter the **Namespaces** value, for example `={"camunda":"http://my.service.com/webservicesserver/"}`.
+
+## Output mapping
+
+### Result variable
+
+You can export a complete response from a SOAP call into a dedicated variable accessible anywhere in a process.
+To do so, input a variable name in the **Result variable** field. Use a unique name to avoid
+overwriting variables.
+
+## Usage examples
```

Comment:
Can we have an example with response mapping as well or is this unusual?

Reply:
Added this section